Backpack/src/androidTest/java/net/skyscanner/backpack/nudger/BpkNudgerTest.kt

```diff
@@ -0,0 +1,89 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.nudger
+
+import android.content.Context
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import net.skyscanner.backpack.R
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.verifyNoInteractions
+
+@RunWith(AndroidJUnit4::class)
+class BpkNudgerTest {
+
+  private lateinit var subject: BpkNudger
+  private lateinit var context: Context
+
+  @Before
+  fun setUp() {
+    context = InstrumentationRegistry.getInstrumentation().targetContext
+    subject = BpkNudger(context)
+  }
+
+  @Test
+  fun test_listenerIsNotInvoked_onSet() {
+    val listener = { _: Int -> }
```

Comment:
Mockito needs the listener the be mocked, e.g. listener = mock<(Int) -> Unit>()

Reply:
Thanks! Turns out I was running the wrong tests and thought they passed 🤦‍♀️